handbook/engineering/roles.md

```diff
@@ -8,13 +8,11 @@ Software engineers build our product and infrastructure.
 
 ### Responsibilities
 
-- Commit to a reasonable amount of work that you are going to get done for each [monthly release](releases/index.md), and then reliably get it done.
-- Post weekly updates about your progress on the [monthly release tracking issue](tracking_issues.md) for your team.
-- Ensure that your work is directly contributing to [our goals](../../company/goals/index.md).
-- Write [RFCs](../communication/rfcs/index.md) to recommend solutions to product and engineering problems.
-- Create and maintain high quality code, tests, and documentation.
-- Support your teammates by reviewing their [code](code_reviews.md) and [RFCs](../communication/rfcs/index.md).
-- Help us build a great team by referring people who you would like to work with again, interviewing candidates, and suggesting improvements to our hiring process.
+- Model [our company values](../../company/values.md) and apply our [guiding engineering principles](index.md#guiding-principles) throughout all your interactions and work.
+- Iteratively create and maintain high quality architecture, code, tests, and documentation that aligns with your team's goals.
+- Document and share your progress in relevant locations at least once a week (for example: in GitHub issues, team slack channel, #progress, [CHANGELOG.md](https://github.com/sourcegraph/sourcegraph/blob/main/CHANGELOG.md)).
+- Unblock and support your teammates by answering questions, reviewing their [code](code_reviews.md), and providing feedback on [RFCs](../communication/rfcs/index.md).
```

Comment:
Is teammate specifically the engineer's team? What about proactively helping other teams with your knoweldge?

Reply:
There are many layers of team and each person is your teammate. There are many things you can do to help your teammates, these are just examples (updating wording to make it clear these are just examples).

```suggestion
- Prioritize unblocking and supporting your teammates (for example: sharing knowledge, answering questions, providing feedback).
```